agent/acl_endpoint_test.go

```diff
@@ -828,6 +828,12 @@ func TestACL_HTTP(t *testing.T) {
 			resp := httptest.NewRecorder()
 			_, err := a.srv.ACLTokenCRUD(resp, req)
 			require.NoError(t, err)
+		})
+		t.Run("Non existent Token", func(t *testing.T) {
+			req, _ := http.NewRequest("GET", "/v1/acl/rules/translate/"+idMap["token-cloned"]+"?token=root", nil)
+			resp := httptest.NewRecorder()
+			_, err := a.srv.ACLRulesTranslateLegacyToken(resp, req)
+			require.Equal(t, NotFoundError{Reason: "No such token exists"}, err)
 			delete(tokenMap, idMap["token-cloned"])
```

Comment:
I think these lines still belong with the previous subtest.

Reply:
I assumed that cloned token that was deleted in the previous test case would make it convenient to be used in this test case.Not an issue if that approach is wrong  I can just use a random ID

Reply:
Oh I see. 

This huge test suffers from some not so great dependencies between subtests but it was written that way to prevent having to have the tests start so many test agents. Other tests further on might expect the cloned token to have been removed so if this test did fail it could produce other false-negative results for the other subtests. So it might be a little better to do the deletions in the other subtest and just hard code some uuid in here (maybe `31de2bf1-0379-43bd-b26a-3f09483b7972`)